facebook/react-native
# chore(releases): rename bump-oss script to align with newer versions

## Summary:



This is a quick PR to align release loop with the newer versions: 71 (https://github.com/facebook/react-native/pull/38890), 72 (https://github.com/facebook/react-native/pull/38888), 73 (https://github.com/facebook/react-native/pull/38666 + https://github.com/facebook/react-native/pull/38887) and onward.

## Changelog:



[INTERNAL] [CHANGED] - rename bump-oss script to align with newer versions

## Test Plan:


* checkout this branch
* comment L54 of trigger-react-native-release.js (to remove the check on the branch name)
* run `yarn trigger-react-native-release -t asd -v asd` (the "fake" parameters are needed to pass the yargs check), verify that it will work as normally expected
